INN/doubleclick-for-wp
# More uninitialized variables, this time in the widget

## Changes

- fixes a handful of uninitialized variables in the widget form and widget frontend
- moves `docs/index.md` to `docs/readme.md` so that people visiting https://github.com/INN/doubleclick-for-wp/tree/master/docs will see the rendered Markup contents of the index
- removes a couple of unnecessary `p` tags in the widget form
- adds a link to breakpoint setting docs from the widget form, see screenshot below

<img width="448" alt="screen shot 2018-10-22 at 11 58 24 pm" src="https://user-images.githubusercontent.com/1754187/47335197-0c804880-d658-11e8-9f0e-6fa7f623f35d.png">


## Why

Continued cleanup, started by #8.

Possibly addresses https://github.com/INN/doubleclick-for-wp/issues/31

## Testing/Questions

Questions that need to be answered before merging:

- [x] Does it work with settings saved in previous iterations of the plugin?

Steps to test this PR:

1. Save widget settings using an old version of this plugin, 0.2.1.
2. Check out this branch
3. Do the widget settings on the form match what they previously were?
4. Do the same as above, but in the Customizer
